examples/aws/api_gateway_with_domain_mtls/aws_gateway_with_domain_mtls.tf

```
@@ -18,7 +18,7 @@ module "lambda_api_gateway_with_domain_mtls" {
     s3_truststore_uri = "s3://they-test-api-gateway-with-domain-assets/certificates/truststore.pem"
     zone_name         = "they-code.de."
     # used domain without timestamp during mtls development, which now delays lambda host resolution (approx. 1hr).
```

```suggestion
    # reusing domains leads to long host resolution delays (approx. 1hr), therefore use a timestamp to create unique domains
```

same within _examples/aws/api_gateway_with_domain/api_gateway_with_domain.tf_